core/controllers/base_test.py

```diff
@@ -1423,3 +1469,404 @@ def test_that_correct_derived_class_does_not_raise_exception(self):
         with self.swap(self, 'testapp', self.mock_testapp):
             self.post_json(
                 '/correctmock', payload, expected_status_int=200)
+
+
+class SchemaValidationIntegrationTests(test_utils.GenericTestBase):
+    """Tests all the functionality of SVS(Schema-Validation-System)
+    architecture.
+    """
+
+    non_schema_handlers = payload_validator.NON_SCHEMA_HANDLERS
+    non_schema_requiring_handlers = (
+            payload_validator.NON_SCHEMA_REQUIRING_HANDLERS)
+    handlers_need_schema = []
+    handlers_to_remove = []
+    handlers_with_missing_url_schema_keys = []
+    handlers_with_missing_req_schema_keys = []
+    handlers_with_non_conforming_default_schema = []
+
+    wiki_page_link = (
+        'https://github.com/oppia/oppia/wiki/Validation-of-handler-args')
+
+    def every_handler_class_has_schema(self, handler, handler_class_name):
```

Comment:
```suggestion
    def _check_every_handler_class_has_schema(self, handler, handler_class_name):
```
Update the docstring accordingly.

Reply:
Done